backend/src/openarchiefbeheer/destruction/templatetags/destruction_list_link.py

```diff
@@ -0,0 +1,24 @@
+from typing import Literal
+
+from django import template
+from django.conf import settings
+
+from furl import furl
+
+from ..models import DestructionList
+
+register = template.Library()
+
+
+@register.simple_tag
+def destruction_list_link(
```

Comment:
I'm wondering if we should move the implementation to the model and use `get_absolute_url` instead. Of course then a solution should be thought to differentiate between the detail page and the review page.

Reply:
As far as I know [`get_absolute_url`](https://docs.djangoproject.com/en/5.1/ref/models/instances/#get-absolute-url) allows you to implement this freely and doesn't need a request? It should be possible to take an optional frontend domain into account. I'm wondering about the page param though and whether it would work with the admin.

Just a heads up, if you see difficulties implementing this we can leave this as is.

Reply:
As far as I know [`get_absolute_url`](https://docs.djangoproject.com/en/5.1/ref/models/instances/#get-absolute-url) allows you to implement this freely and doesn't need a request? It should be possible to take an optional frontend domain into account. I'm wondering about the page param though and whether it would work with the admin.

Just a heads up, if you see difficulties implementing this we can leave this as is.

Reply:
Ahh in my head I had this one: https://docs.djangoproject.com/en/5.1/ref/request-response/#django.http.HttpRequest.build_absolute_uri
But you meant this one I think: https://docs.djangoproject.com/en/5.1/ref/models/instances/#get-absolute-url

To pass the parameter it would still need to be a template tag though :thinking:

Reply:
Now that we changed that we only include the list name, it is actually not possible to make it a method on the model (because we only have the name). So I will leave this a templatetag!